automated/linux/leds/test-leds2.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+source "../test_format.sh"
+
+LED_TIME=1 #.5
```

Comment:
what is the meaning of .5 ?

Reply:
LED_TIME is the time in seconds the LED is on or off.
In that case 0.5 equals a time of half a second.
I will give a description of the variable.